ClassicAssist/Data/Macros/Commands/JournalCommands.cs

```diff
@@ -6,33 +6,44 @@
 using ClassicAssist.Shared.Resources;
 using ClassicAssist.UO.Data;
 using ClassicAssist.UO.Network;
+using ClassicAssist.UO.Objects;
 
 namespace ClassicAssist.Data.Macros.Commands
 {
     public static class JournalCommands
     {
-        [CommandsDisplay( Category = nameof( Strings.Journal ),
-            Parameters = new[]
```

Comment:
Put all these parameters back, they're used in the wiki docs

Reply:
Done